### PR TITLE
Includes Bascinet Recipes FINALLY

### PR DESCRIPTION
**Pull request by WWWWW-Git**
Implements the Frog Helmet and Slit Kettle helmet in game as well as the Bascinet and Decorated Bascinet
frog
kettle

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 1]
```diff
@@ -248,6 +248,22 @@
 	createditem_num = 1
 	craftdiff = 2
 
+/datum/anvil_recipe/armor/helmetslitkettle
+	name = "Slitted Kettle helmet x2"
+	recipe_name = "a pair of Slitted Kettle Helmets"
```
Comment (lavillastrangiato) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:253]: Should be "A pair of slitted kettle helmets."

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 2]
```diff
@@ -248,6 +248,22 @@
...
+	req_bar = /obj/item/ingot/steel
+	created_item = /obj/item/clothing/head/roguetown/helmet/slitkettle
+	createmultiple = TRUE
+	createditem_num = 1
+	craftdiff = 2
+
+/datum/anvil_recipe/armor/froghelmet
+	name = "Frog helmet"
+	recipe_name = "a Frog Helmet"
```
Comment (lavillastrangiato) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:262]: "A frog helmet."

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 3]
```diff
@@ -320,6 +336,13 @@
 	created_item = /obj/item/clothing/head/roguetown/helmet/sallet
 	craftdiff = 2
 
+/datum/anvil_recipe/armor/bascinet
+	name = "Bascinet"
```
Comment (lavillastrangiato) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:340]: "Bassinet", not bascinet.

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 4]
```diff
@@ -320,6 +336,13 @@
...
+	recipe_name = "a Bascinet"
```
Comment (lavillastrangiato) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:341]: Ditto my last.

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 5]
```diff
@@ -342,28 +365,32 @@
...
 	req_bar = /obj/item/ingot/steel
-	additional_items = list(/obj/item/ingot/steel,/obj/item/natural/cloth)
+	additional_items = list(/obj/item/ingot/steel,/obj/item/ingot/gold,/obj/item/natural/cloth)
 	created_item = /obj/item/clothing/head/roguetown/helmet/heavy/decorated/golden
 	craftdiff = 3
-	i_type = "Armor"
+
+/datum/anvil_recipe/armor/decoratedbascinet
+	name = "Decorated Bascinet (c)"
+	recipe_name = "a Decorated Bascinet"
```
Comment (lavillastrangiato) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:392]: x3 combo.

**Comment by NPC1314** [comment]
Bascinet is also a correct way to spell it, and its been used in the game for a long time AFAIK so better just keep using bascinet.

File: code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm [hunk 6]
```diff
@@ -342,28 +365,32 @@
 	additional_items = list(/obj/item/ingot/steel,/obj/item/natural/cloth)
 	created_item = /obj/item/clothing/head/roguetown/helmet/heavy/decorated/knight
 	craftdiff = 4
-	i_type = "Armor"
```
Comment (NPC1314) [inline code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/armor.dm:345]: Whats this about?

**Review by NPC1314 — APPROVED** [review]
Froghelm onmob issues remain and a couple question marks about those recipes but whatever.

**Pull request merged**